templates/_visitorDetails.twig

```diff
@@ -3,7 +3,7 @@
     {% for customDimension in customDimensions if customDimension.value|length > 0 %}
         <br/>
         <abbr class="visitorLogTooltip" title="{{ 'CustomDimensions_CustomDimensionId'|translate(customDimension.id) }}">{{ customDimension.name|truncate(30) }}:</abbr>
-        {{ customDimension.value|truncate(50) }}
+        {{ customDimension.value|truncate(50)|rawSafeDecoded }}
```

Comment:
This is not an optimal solution, as truncate uses the encoded string, which might cut the string to early or even within an encoded char.
Switching the filter order doesn't work, as the truncate filter would mark the value as "not safe" again and it would be encoded again.
I guess we would need to adjust the truncate filter to respect encoded entities to fix that correct.

Reply:
Would `customDimension.value|rawSafeDecoded|truncate(50)|raw` work and be safe?

Reply:
Hi @sgiehl could you reply to @diosmosis comment when you get a chance?

Reply:
ping @sgiehl

Reply:
@mattab just had a look at that one again. It's been a long time and it seems the `|truncate(50)|rawSafeDecoded` now works as expected. Truncate now seems to fully respect encoded entities. Guess we can merge as is now.